GitUI/Shells/BashShell.cs

```diff
@@ -18,16 +19,31 @@ public BashShell()
             Name = ShellName;
             Icon = Images.GitForWindows;
 
-            ExecutableName = BashExe;
-            if (PathUtil.TryFindShellPath(ExecutableName, out var exePath))
+            if (PathUtil.TryFindShellPath(GitBashExe, out var exePath))
             {
+                ExecutableName = GitBashExe;
                 ExecutablePath = exePath;
+                //// do not set ExecutableCommandLine: git-bash.exe cannot be connected to built-in console
+            }
+
+            if (PathUtil.TryFindShellPath(BashExe, out exePath))
```

Comment:
?
```suggestion
            else if (PathUtil.TryFindShellPath(BashExe, out exePath))
```

Reply:
No `else if` on purpose.

Should I reformulate the above comment to
```
//// Do not set ExecutableCommandLine to git-bash.exe since it cannot be connected to the built-in console.
//// For the console, try to find bash or sh below.
```
?

Reply:
Do not use ////, should only be used for TODO and other that should be changed, add the extra newline

Maybe?
```c#

// Try to find bash or sh below to set ExecutableCommandLine, as git-bash.exe cannot be connected to the built-in console.
```

Reply:
Thank you, I took this.